Tests/CommonTests/AssemblyResolverTests.cs

```diff
@@ -22,6 +22,19 @@ public class AssemblyResolverTests
 
         #region Tests
 
+        [TestMethod]
+        public void AssemblyResolver_Creation()
+        {
+            // 1. Null logger
+            AssertException.Expect<ArgumentNullException>(() => new AssemblyResolver(null, new string[] { this.TestContext.TestDeploymentDir }));
+
+            // 2. Null paths
+            AssertException.Expect<ArgumentException>(() => new AssemblyResolver(new TestLogger(), null));
+
+            // 3. Empty paths
+            AssertException.Expect<ArgumentException>(() => new AssemblyResolver(new TestLogger(), new string[] { }));
+        }
+
```

Comment:
These are pretty low-value tests and I wouldn't normally write them for every class. However, I only found the most recent error by looking at the code coverage for the file and spotting a line expected to be covered wasn't. I added this test as it should take the coverage up to 100% as theses were the only other uncovered lines i.e. it'll make it simpler to see if the other cases have been covered without having to drill in to file.